source/fe/mapping_q_cache.cc

```diff
@@ -77,7 +77,23 @@ MappingQCache<dim, spacedim>::initialize(
   const MappingQGeneric<dim, spacedim> &mapping)
 {
   AssertDimension(this->get_degree(), mapping.get_degree());
+  initialize(triangulation,
+             [&mapping](const typename Triangulation<dim>::cell_iterator &cell)
+               -> std::vector<Point<spacedim>> {
+               return mapping.compute_mapping_support_points(cell);
```

Comment:
This is a case where things would actually have been shorter with `std::bind` :-) But we're not going to bring that back...

Reply:
Yes, I would have complained. :wink: